openprescribing/measure_definitions/opioidper1000.json

```diff
@@ -34,30 +34,32 @@
-    "0407020AD%AP # Oxycodone HCl_Tab 120mg M/R (brands and generic)",
-    "0407020AD%AQ # Oxycodone HCl_Tab 60mg M/R (brands and generic)",
-    "0407020AF%AD # Oxycodone HCl/NaloxoneHCl_Tab 40/20mgM/R (brands and generic)",
-    "0407020AG%AF # Tapentadol HCl_Tab 200mg M/R (brands and generic)",
-    "0407020AG%AG # Tapentadol HCl_Tab 250mg M/R (brands and generic)",
-    "0407020B0%AF # Buprenorphine_Patch 52.5mcg/hr (96hr) (brands and generic)",
-    "0407020B0%AG # Buprenorphine_Patch 70mcg/hr (96hr) (brands and generic)",
-    "0407020Q0%AG # Morph Sulf_Tab 200mg M/R (brands and generic)",
-    "0407020Q0%AH # Morph Sulf_Tab 100mg M/R (brands and generic)",
-    "0407020Q0%AI # Morph Sulf_Tab 60mg M/R (brands and generic)",
-    "0407020Q0%DC # Morph Sulf_Gran Sach 60mg M/R S/F (brands and generic)",
-    "0407020Q0%DD # Morph Sulf_Gran Sach 100mg M/R S/F (brands and generic)",
-    "0407020Q0%DE # Morph Sulf_Gran Sach 200mg M/R S/F (brands and generic)",
-    "0407020Q0%DU # Morph Sulf_Cap 90mg M/R (brands and generic)",
-    "0407020Q0%DV # Morph Sulf_Cap 120mg M/R (brands and generic)",
-    "0407020Q0%DW # Morph Sulf_Cap 150mg M/R (brands and generic)",
-    "0407020Q0%EB # Morph Sulf_Cap 100mg M/R (brands and generic)",
-    "0407020Q0%EH # Morph Sulf_Cap 60mg M/R (brands and generic)",
-    "0407020Q0%EI # Morph Sulf_Cap 200mg M/R (brands and generic)"
+    "0407020A0%AF     # Fentanyl_Transdermal Patch 50mcg/hr (brands and generic)",
+    "0407020A0%AG     # Fentanyl_Transdermal Patch 75mcg/hr (brands and generic)",
+    "0407020A0%AH     # Fentanyl_Transdermal Patch 100mcg/hr (brands and generic)",
+    "0407020AD%AH     # Oxycodone HCl_Tab 40mg M/R (brands and generic)",
+    "0407020AD%AI     # Oxycodone HCl_Tab 80mg M/R (brands and generic)",
+    "0407020AD%AP     # Oxycodone HCl_Tab 120mg M/R (brands and generic)",
+    "0407020AD%AQ     # Oxycodone HCl_Tab 60mg M/R (brands and generic)",
+    "0407020AF%AD     # Oxycodone HCl/NaloxoneHCl_Tab 40/20mgM/R (brands and generic)",
+    "0407020AG%AF     # Tapentadol HCl_Tab 200mg M/R (brands and generic)",
+    "0407020AG%AG     # Tapentadol HCl_Tab 250mg M/R (brands and generic)",
+    "0407020B0%AF     # Buprenorphine_Patch 52.5mcg/hr (96hr) (brands and generic)",
+    "0407020B0%AG     # Buprenorphine_Patch 70mcg/hr (96hr) (brands and generic)",
+    "0407020Q0%AG     # Morph Sulf_Tab 200mg M/R (brands and generic)",
+    "0407020Q0%AH     # Morph Sulf_Tab 100mg M/R (brands and generic)",
+    "0407020Q0%AI     # Morph Sulf_Tab 60mg M/R (brands and generic)",
+    "0407020Q0%DC     # Morph Sulf_Gran Sach 60mg M/R S/F (brands and generic)",
+    "0407020Q0%DD     # Morph Sulf_Gran Sach 100mg M/R S/F (brands and generic)",
+    "0407020Q0%DE     # Morph Sulf_Gran Sach 200mg M/R S/F (brands and generic)",
+    "0407020Q0%DU     # Morph Sulf_Cap 90mg M/R (brands and generic)",
+    "~0407020Q0BIACDU # MXL Cap 60mg",
```

Comment:
added in as an exclusion so that MXL no longer appears in the analysis.

Reply:
Markdown should be `MXL 90mg`